plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1915,3 +1915,14 @@ SecRule REQUEST_FILENAME "@rx /apps/mail/api/outbox/(?:from-draft/)?[0-9]+$" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.body,\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.editorBody,\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.subject"
+
+# Moving a card in Deck application
+SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+/reorder?$" \
+    "id:9508993,\
```

Comment:
You've placed this rule underneath the Nextcloud Mail group, you should place Nextcloud Deck rule exclusions under it's own group. It's easier to understand what a group of rule does at a glance vs having to look into detail to each rule.

Reply:
That's better, but you should use a different rule ID. There's only about 7 rule IDs left now. There's lots of free space if you look before the Users group, about 40 rule IDs free if you use ``9508810``. I doubt this will be the only false positive with Deck, knowing Nextcloud.

Reply:
I did change the rule ID